.rhcicd/clowdapp-backend.yaml

```diff
@@ -198,7 +198,7 @@ parameters:
 - name: NOTIFICATIONS_LOG_LEVEL
   description: Log level for com.redhat.cloud.notifications
   value: INFO
-- name: RBAC_AUTHENTICATION_READ_TIMEOUT
+- name: RBAC_AUTHENTICATION_READ_TIMEOUTIT_USER_SERVICE_REST_URL
```

Comment:
Just bumped on this accidentally, that sounds fishy ?

Reply:
indeed. fixed. thanks!